app/src/main/java/com/ntnu/wip/nabl/MVCView/ExportView/ExportView.java

```diff
@@ -212,7 +212,7 @@ public void setResourceViewerAdapter(Adapter adapter) {
     public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
         final String date = String.format(Locale.getDefault(), "%s.%s.%s",
                 String.valueOf(dayOfMonth),
-                String.valueOf(month),
+                String.valueOf(month + 1),                  // +1 because the month is 0-based Array
```

Comment:
because month is defined by Calendar constants which states that January is 0

Reply:
But the logic is right ... you mean to change the because comment ?
The string that will be shown to the user is incorrect

Reply:
Yeah, just the comment since it is misleading as of now

Reply:
I'll change it for you